jockeyJSAndroid/build.gradle

```diff
@@ -0,0 +1,22 @@
+apply plugin: 'com.android.library'
```

Comment:
As long as we don't end up with two separate aars after running `gradle build` then we should be ok. Won't you check whether or not that is the case, @tomerlevi444?